frontend/src/components/explorer/MapViewer.vue

```diff
@@ -114,30 +114,14 @@
                 }"
               ></i>
             </span>
-            <br />
-            D
-            <br />
-            A
-            <br />
-            T
-            <br />
-            A
-            <br />
-            <br />
-            O
-            <br />
-            V
-            <br />
-            E
-            <br />
-            R
-            <br />
-            L
-            <br />
-            A
-            <br />
-            Y
-            <br />
+            <span
+              v-for="(c, i) in 'DATA OVERLAY'"
+              :key="i"
+              :class="c === ' ' ? 'mt-4' : ''"
+              class="is-block"
+            >
+              {{ c }}
+            </span>
```

Comment:
Just wanted to praise that this is a _much_ better way of vertically arranging the letters than before.